modules/widgets/milestone/milestone.php

```diff
@@ -213,20 +197,28 @@ function widget( $args, $instance ) {
 		echo '<span class="date">' . esc_html( date_i18n( __( 'F jS, Y', 'jetpack' ), $milestone ) ) . '</span>';
 		echo '</div>';
 
-		if ( 1 > $diff ) {
-			/* Milestone has past. */
+		if ( ( 1 > $diff ) && ( 'until' === $type ) ) {
 			echo '<div class="milestone-message">' . $instance['message'] . '</div>';
 		} else {
+			if ( 'since' === $type ) {
+				$text = __( 'ago.', 'jetpack' );
```

Comment:
I wonder if this may prove hard to translate without context, or in some languages.

Reply:
We can address this in a further PR if necessary